source/catalog.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html class="page" lang="en">
```

Comment:
Сайт все-таки на русском)
```suggestion
<html class="page" lang="ru">
```